mappings/net/minecraft/server/BannedPlayerEntry.mapping

```diff
@@ -7,5 +9,5 @@ CLASS net/minecraft/class_3336 net/minecraft/server/BannedPlayerEntry
 		ARG 3 source
 		ARG 4 expiry
 		ARG 5 reason
-	METHOD method_14651 getProfileFromJson (Lcom/google/gson/JsonObject;)Lcom/mojang/authlib/GameProfile;
+	METHOD method_14651 profileFromJson (Lcom/google/gson/JsonObject;)Lcom/mojang/authlib/GameProfile;
```

Comment:
Why not `get-`?

Reply:
this is converting this json object to a profile than getting a profile as part of this json object

Reply:
But if it's `getProfile(JsonObject)` it's clear it's getting it from JSON too.

Reply:
nah getProfile is more like 
```json
{
  "profile": {(the_stuff_you_want_to_get)}
}
```
but actually it is
```json
{(the_stuff_you_want_to_get)}
```
So a better name is `profileFromJson` or `jsonToProfile`, whichever you like @Runemoro